src/Common/tests/Collections/ICollectionTest.cs

```diff
@@ -299,6 +300,19 @@ public void CopyToValidTypes()
             }
         }
 
+        [Fact]
+        public void CollectionShouldContainAllItems()
+        {
+            object[] items = GenerateItems(16);
+            ICollection<T> collection = GetCollection(items) as ICollection<T>;
+            if (collection == null)
+                return;
```

Comment:
The collection type is not required to implement generic ICollection. I didn't want to assume that it would.